components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/io/zip.ts

```diff
@@ -12,84 +12,93 @@
 /**
  * API Zip
  */
-import * as bytes from "@dirigible/io/bytes";
+import {bytes} from ".";
+import { InputStream, OutputStream } from "./streams";
 const ZipFacade = Java.type("org.eclipse.dirigible.components.api.io.ZipFacade");
 
-export function zip(sourcePath, zipTargetPath) {
+export function zip(sourcePath: string, zipTargetPath: string): void {
     ZipFacade.exportZip(sourcePath, zipTargetPath);
 };
-export function unzip(zipPath, targetPath) {
+export function unzip(zipPath: string, targetPath: string): void {
     ZipFacade.importZip(zipPath, targetPath);
 };
 
-export function createZipInputStream(inputStream) {
+export function createZipInputStream(inputStream: InputStream): ZipInputStream {
     const native = ZipFacade.createZipInputStream(inputStream.native);
     return new ZipInputStream(native);
 };
 
 class ZipInputStream {
 
-    constructor(private native) {}
+    private readonly native: any
 
-    getNextEntry() {
+    constructor(native: any) {
+        this.native = native;
+    }
+
+    public getNextEntry(): ZipEntry {
         const native = this.native.getNextEntry();
         const zipEntry = new ZipEntry(native);
         return zipEntry;
     };
 
-    read() {
+    public read(): Array<bytes> {

```

Comment:
Why explicitly use `public`? Also, `Array<bytes>` seems incorrect, the type should be `Byte[]` where `Byte` is https://docs.oracle.com/javase/8/docs/api/java/lang/Byte.html You should probably create a custom type inside the `bytes` module as 
```TS
export type Byte = typeof Java.type('java.lang.Byte');
```

Reply:
Same goes for other similar places